pkg/features/kube_features.go

```diff
@@ -115,6 +115,12 @@ const (
 	//
 	// New local storage types to support local storage capacity isolation
 	LocalStorageCapacityIsolation utilfeature.Feature = "LocalStorageCapacityIsolation"
+
+	// owner: @ilackams
+	// alpha: v1.7
+	//
+	// Enables compression of REST responses (GET and LIST only)
+	APIResponseCompression utilfeature.Feature = "APIResponseCompression"
```

Comment:
This is actually `staging/src/k8s.io/apiserver/pkg/features/kube_features.go`.  It seems unusual, but the gate is actually generic since your changes were generic.